### ISSUE TITLE
vmstorage: excessive logging when switching the storage to read-only mode

### ISSUE THREAD

**Issue by hy-ntyni** [VictoriaMetrics/VictoriaMetrics#5159]
### Is your question request related to a specific component?

vmstorage

### Describe the question in detail

Hi, I'm testing vmstorage cluster mode behaviour when the disk fills up, and it looks like vmstorage will log a message every second when storage.minFreeDiskSpaceBytes is passed. Is this intentional?

(I've raised -storage.minFreeDiskSpaceBytes to 800MB here just for testing.)
```
  2023-10-10T23:57:51.093Z        warn    VictoriaMetrics/lib/storage/storage.go:669      switching the storage at /vmstorage-data to read-only mode, since it has less than -storage.minFreeDiskSpaceBytes=838860800 of free space: 838803456 bytes left
  2023-10-10T23:57:52.093Z        warn    VictoriaMetrics/lib/storage/storage.go:669      switching the storage at /vmstorage-data to read-only mode, since it has less than -storage.minFreeDiskSpaceBytes=838860800 of free space: 838803456 bytes left
  2023-10-10T23:57:53.092Z        warn    VictoriaMetrics/lib/storage/storage.go:669      switching the storage at /vmstorage-data to read-only mode, since it has less than -storage.minFreeDiskSpaceBytes=838860800 of free space: 838803456 bytes left
[...]
  2023-10-11T09:20:20.092Z        warn    VictoriaMetrics/lib/storage/storage.go:669      switching the storage at /vmstorage-data to read-only mode, since it has less than -storage.minFreeDiskSpaceBytes=838860800 of free space: 838692864 bytes left
  2023-10-11T09:20:21.093Z        warn    VictoriaMetrics/lib/storage/storage.go:669      switching the storage at /vmstorage-data to read-only mode, since it has less than -storage.minFreeDiskSpaceBytes=838860800 of free space: 838692864 bytes left
  2023-10-11T09:20:22.093Z        warn    VictoriaMetrics/lib/storage/storage.go:669      switching the storage at /vmstorage-data to read-only mode, since it has less than -storage.minFreeDiskSpaceBytes=838860800 of free space: 838692864 bytes left
  2023-10-11T09:20:23.094Z        warn    VictoriaMetrics/lib/storage/storage.go:669      switching the storage at /vmstorage-data to read-only mode, since it has less than -storage.minFreeDiskSpaceBytes=838860800 of free space: 838692864 bytes left
  2023-10-11T09:20:24.094Z        warn    VictoriaMetrics/lib/storage/storage.go:669      switching the storage at /vmstorage-data to read-only mode, since it has less than -storage.minFreeDiskSpaceBytes=838860800 of free space: 838692864 bytes left
```

I was thinking of letting the vmstorage instances fill up and then just remove them from the vminsert backend list but keep them on the vmselect list. This logging rate makes that unsustainable. And repeating the same thing over and over does not seem very useful?

Tested with v1.87.9 and v1.93.5.

### Troubleshooting docs

- [X] General - https://docs.victoriametrics.com/Troubleshooting.html
- [X] vmagent - https://docs.victoriametrics.com/vmagent.html#troubleshooting
- [X] vmalert - https://docs.victoriametrics.com/vmalert.html#troubleshooting

**Comment by hagen1778** [comment]
Thanks for reporting this issue! I hope it will be addressed at https://github.com/VictoriaMetrics/VictoriaMetrics/pull/5191

**Comment by valyala** [comment]
`vmstorage` should emit only a single log line when it switches to [read-only mode](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html#readonly-mode), starting from [v1.93.7 LTS release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.93.7). Closing the issue as fixed.

**Comment by valyala** [comment]
FYI, the fix has been included in [v1.87.11 LTS release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.87.11).

**Comment by valyala** [comment]
FYI, the fix has been included in [v1.95.0 release](https://docs.victoriametrics.com/CHANGELOG.html).
